### PR TITLE
Add YouTrack recipe

### PR DESCRIPTION
**Pull request by radamanthus**

### CODE REVIEW

File: cookbooks/docker_youtrack/templates/default/youtrack.conf.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+server {
+  listen 8081;
```
Comment (crigor) [inline cookbooks/docker_youtrack/templates/default/youtrack.conf.erb]: Let's use port 80.

File: cookbooks/docker_youtrack/recipes/setup_nginx.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+docker_instance = instances.find{|i| i[:name] == node[:docker_youtrack][:utility_name]}
+if docker_instance && (['app_master', 'app', 'solo'].include? node.dna[:instance_role])
+  template '/etc/nginx/servers/youtrack.conf' do
+    owner 'deploy'
+    group 'deploy'
+    mode 0644
+    source 'youtrack.conf.erb'
+    notifies :reload, 'service[nginx]'
+    variables ({
+      server_name: 'youtrack.mydomain.com',
```
Comment (crigor) [inline cookbooks/docker_youtrack/recipes/setup_nginx.rb]: Let's put this in attributes.

File: cookbooks/docker_youtrack/templates/default/youtrack.conf.erb (hunk 1 above)
Comment (radamanthus) [inline cookbooks/docker_youtrack/templates/default/youtrack.conf.erb]: port 80 on both container and host ports?

Comment (crigor) [inline cookbooks/docker_youtrack/templates/default/youtrack.conf.erb]: Port 80 on the nginx config on the app instances.

Comment (crigor) [inline cookbooks/docker_youtrack/templates/default/youtrack.conf.erb]: Scratch that. Nginx uses port 8081 not 81.

File: cookbooks/docker_youtrack/templates/default/youtrack.conf.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  server_name <%= @server_name %>;
+
+  location / {
+    proxy_pass http://<%= @docker_youtrack_hostname %>:9090;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host $http_host;
+    proxy_set_header X-Forwarded-Proto https;
```
Comment (crigor) [inline cookbooks/docker_youtrack/templates/default/youtrack.conf.erb]: Remove this as we're not using https on this example.

**Pull request merged**